vmdb/app/models/miq_server/role_management.rb

```diff
@@ -67,9 +67,9 @@ def set_role_activation(active, *roles)
     roles = roles.first if roles.length == 1 && roles[0].kind_of?(Array)
     return if roles.empty?
 
-    roles = (roles.length == 1 && roles[0] == "*") ? self.server_roles : ServerRole.find_all_by_name(roles)
+    roles = (roles.length == 1 && roles[0] == "*") ? self.server_roles : ServerRole.where(:name => roles)
     ids   = roles.collect { |r| r.id }
```

Comment:
Is the `roles` variable used elsewhere aside from the .collect here?  If not, should the query(ies) have `.select(:id)` or `.pluck(:id)`?